bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -37,49 +37,6 @@ spec:
       {{- $postgresqlHeadlessServiceName := printf "%s-headless" (include "postgresql-ha.postgresql" .) }}
       {{- $releaseNamespace := .Release.Namespace }}
       {{- $clusterDomain:= .Values.clusterDomain }}
-      initContainers:
```

Comment:
If even a single backend node is unable to come online the postgresql-ha deployment will not be ready to accept connections. We should not have to wait for backend nodes to come online in the pgpool deployment, ideally pgpool would manage the connections. Therefore i've removed this initContainer